osidb/exception_handlers.py

```diff
@@ -23,4 +24,12 @@ def exception_handler(exc, context):
 
     if isinstance(exc, DjangoValidationError):
         exc = DRFValidationError(as_serializer_error(exc))
+
+    if isinstance(exc, OperationalError):
+        data = {
+            "detail": "[OperationalError] Concurrent access to the same model by you and someone else. For example, two clients editing the same Flaw. Exception details: "
+            + str(exc)
```

Comment:
I am not sure if every `OperationalError` is caused by concurrent access. Can we not use only output from `exc` instead of creating our own?

Reply:
We could check the error message for some keyword (deadlock?) to be certain that it is the concurrent access issue. Otherwise it would probably be an implementation error so re-raising might be OK.

Reply:
Good point. Turns out it can happen with other DB errors too. Thanks! Fixed.